.github/workflows/pytest.yml

```diff
@@ -35,12 +35,14 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r devtools/dev-requirements.txt
+          pip install matplotlib==3.5.0
```

Comment:
Why `3.5.0` here if mpl tests uses up to `3.6.0`?

Reply:
mpl 3.6.0 has a minimum python version of 3.8, while we still run tests on 3.7 (for now). The point here is just to pin to (any) reasonable version of mpl for testing consistency, with the assumption that different mpl versions will all work with possible minor differences in plot alignment etc.